packages/ckeditor5-image/src/imageupload/imageuploadediting.ts

```diff
@@ -135,6 +137,14 @@ export default class ImageUploadEditing extends Plugin {
 
 				editor.execute( 'uploadImage', { file: images } );
 			} );
+
+			const uploadImageCommand = editor.commands.get( 'uploadImage' )!;
+			const t = this.editor.locale.t;
+
+			if ( !uploadImageCommand.isAccessAlowed ) {
+				window.alert( t( 'No permission to upload from computer. Try to use file ' + // eslint-disable-line no-alert
```

Comment:
Will it work with our tooling if the string is dynamically created (`+` operator)?

Reply:
I mean, we statically scan for `t` invocations to find all the translations we use. It's argument should be literal string.

Reply:
You should use our notification feature same as CKFinder integration uses it:

https://github.com/ckeditor/ckeditor5/blob/08270aafcc6e7e4fb4f4a4d2ebbc7451f031d303/packages/ckeditor5-ckfinder/src/ckfindercommand.ts#L146-L152